packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -0,0 +1,193 @@
+import type { ReactElement } from 'react';
+
+import {
+  currencyAbbreviationsMapping,
+  currencylocaleMapping,
+  currencyPrefixMapping,
+} from './amountTokens';
+import AmountValue, { getAffixFontSize } from './AmountValue';
+import { StyledAmount } from './StyledAmount';
+import { BaseText } from '~components/Typography/BaseText';
+import type { Feedback } from '~tokens/theme/theme';
+import type { BaseTextProps } from '~components/Typography/BaseText/types';
+import BaseBox from '~components/Box/BaseBox';
+import type { TestID } from '~src/_helpers/types';
+import { metaAttribute, MetaConstants } from '~utils';
+
+export const suffixTypes = {
+  HUMANIZE: 'humanize',
+  DECIMALS: 'decimals',
+  NONE: 'none',
+};
+
+type currencyType = 'INR' | 'MYR';
+
+type AmountProps = {
+  /**
+   * The value to be rendered within the component.
+   *
+   */
+  value: number;
+  /**
+   * Sets the intent of the amount.
+   *
+   * @default 'neutral'
+   */
+  intent?: Feedback | undefined;
+  /**
+   * Sets the variant of the amount.
+   *
+   * @default 'heading'
+   */
+  size?:
+    | `title-medium`
+    | `title-small`
+    | `heading-large`
+    | `heading-large-bold`
+    | `heading-small`
+    | `heading-small-bold`
+    | `body-medium`
+    | `body-medium-bold`
+    | `body-small`
+    | `body-small-bold`;
```

Comment:
Seems like we are missing few sizes, body-xsmall, heading-medium etc 🤔 

We should compose this union from the title/heading/body props, otherwise they might get out of sync.